api/src/services/report/smsparser.js

```diff
@@ -187,7 +187,7 @@ exports.parseField = (field, raw) => {
       if (raw === undefined) {
         return;
       }
-      const val = parseNum(raw);
+      const val = parseNum(raw); /* eslint-disable-line no-case-declarations */
```

Comment:
Did you read the rules about this one and decide that we wanted to override it? If ESLint is finding problems we should consider each one and see if we want to override or solve.

Reply:
Damn that _Travis_ guy!

Can you please raise an issue for each line we should fix but you've ignored for now? Otherwise we'll probably never remember to fix them...